src/Components/card/Card.css

```diff
@@ -0,0 +1,105 @@
+.modal-container {
+  display: flex;
+  margin-left: 2rem;
+  margin-right: 2rem;
+  margin-bottom: 2rem;
+  justify-self: center;
+  width: 80%;
+  flex-direction: column;
+}
+.heading {
+  align-self: center;
+}
+
+.modal-flex-container {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  height: 75vh;
+  width: 35%;
+  margin-top: 0rem;
+  background-color: white;
+  box-shadow: 1.5px 1.5px 12px 0px lightgray;
+  margin-bottom: 2rem;
+}
+
+.image {
+  align-self: center;
+  height: 70%;
+  width: 93%;
+  margin: 1px;
+}
+
+.eng {

```

Comment:
I would suggest changing "eng", "slim" class names, it does not seem descriptive.